#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("max_distance", double_t, 0, "the distance to reach the color to the maximum color", 0.1, 0.0, 2.0)
gen.add("min_distance", double_t, 0, "the distance to reach the color to the minimum color", 0.0, 0.0, 2.0)
gen.add("only_projectable", bool_t, 0, "draw the points only if they can be projected inside of convex", False)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "ColorizeDistanceFromPlane"))
